schema/defs-linux.json

```diff
@@ -93,7 +93,9 @@
             "type": "string"
         },
         "FileType": {
-            "type": "integer"
+            "description": "Type of a block or special character device",
+            "type": "string",
+            "pattern": "^[c|b|u|p]$"
```

Comment:
Is this JSON Schema following [the RFC draft 4][1]?  I don't see anything about `pattern` in there, but would expect a regexp to look like `^[cbup]$`.

[1]: https://tools.ietf.org/html/draft-zyp-json-schema-04